app/src/main/java/com/hfad/beeradviser/FindBeerActivity.java

```diff
@@ -28,9 +28,10 @@ public void onClickFindBeer(View view) {
         String beerType = String.valueOf(colorSpinnerView.getSelectedItem());
 
         //TODO: WRITE code to get recommendations from the BeerExpert class
-
+        expert.setColor(beerType);
+        String brands = expert.getRecommendation(expert.getColor());
```

Comment:
Refactor the getRecommendation() to use mColor, since expert already has it's color set in line 31